README.md

```diff
@@ -967,6 +973,10 @@ The pipeline operation for this sink could look like this:
       
       # Optional info version (default = 1)
       info.version = 1
+      
+      # Optional for running Enceladus from Pramen
```

Comment:
It is not optional for Enceladus. But for the setup of Pramen. Right?

```suggestion
      # Optional when running Enceladus from Pramen
```

Or is it 

```suggestion
      # Mandatory when running Enceladus from Pramen
```
?